src/NzbDrone.Core/Notifications/Webhook/WebhookPayload.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.Collections.Generic;
+
+namespace NzbDrone.Core.Notifications.Webhook
+{
+    internal class WebhookPayload
+    {
+        public string EventType { get; internal set; }
```

Comment:
I think it's ok to make these `public` we generally try to avoid using `internal`, 

internal is normally used for libraries where you share your code and you don't want 3rd parties to access certains parts of it, which isn't the case with us.